src/plugins/intel_cpu/tests/functional/single_layer_tests/classes/eltwise.cpp

```diff
@@ -190,7 +191,7 @@ const std::vector<ngraph::helpers::EltwiseTypes>& eltwiseOpTypesBinInp() {
 const std::vector<ngraph::helpers::EltwiseTypes>& eltwiseOpTypesDiffInp() {
         static const std::vector<ngraph::helpers::EltwiseTypes> eltwiseOpTypesDiffInp = { // Different number of input nodes depending on optimizations
         #if defined(OPENVINO_ARCH_X86) || defined(OPENVINO_ARCH_X86_64)
-                ngraph::helpers::EltwiseTypes::POWER,                   //TODO: Fix CVS-111880
+                ngraph::helpers::EltwiseTypes::POWER,
```

Comment:
POWER is still tested only on x86

Reply:
@dmitry-gorokhov this was disabled because of the executor check (now some tests failed because ref != acl), @alvoron can you please describe how to return the ref check back on the implementation